core/src/main/resources/hudson/PluginManager/check.jelly

```diff
@@ -22,12 +22,21 @@ OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 THE SOFTWARE.
 -->
 <?jelly escape-by-default='true'?>
-<j:jelly xmlns:j="jelly:core" xmlns:f="/lib/form">
+<j:jelly xmlns:j="jelly:core" xmlns:f="/lib/form" xmlns:l="/lib/layout">
 
   ${%lastUpdated(app.updateCenter.lastUpdatedString)}
   <f:link href="checkUpdatesServer" post="true" clazz="yui-button yui-submit-button submit-button primary">
     ${%Check now}
   </f:link>
+  <j:if test="${isUpdates}">
```

Comment:
What is this test used for? If there are no updates available, the buttons are shown nevertheless:
![](https://i.imgur.com/mecr5Ly.png)

Reply:
"Available" view will not show this button. If we don't add this test, it will show on the "Available" view.